app/cypress/integration/cif/project-revision/undo-changes.spec.js

```diff
@@ -54,11 +54,11 @@ describe("when undoing, the project revision page", () => {
     cy.fillFundingAgreementForm(
```

Comment:
I didn't add any IA-specific tests to this file because we've got a ticket about refactoring this test into a jest one instead: https://github.com/bcgov/cas-cif/issues/876